besu/src/main/java/org/hyperledger/besu/cli/options/unstable/EthProtocolOptions.java

```diff
@@ -34,6 +34,8 @@ public class EthProtocolOptions implements CLIOptions<EthProtocolConfiguration>
   private static final String LEGACY_ETH_64_FORK_ID_ENABLED =
       "--compatibility-eth64-forkid-enabled";
 
+  private static final String MAX_CAPABILITY_ENABLED = "--Xeth-max-capability-enabled";
```

Comment:
re the name - generally the "enabled" suffix indicates a boolean 
what about
--X-eth-capability-max

Reply:
Done